### DISCUSSION TITLE
could not find function "addArchRLocking"

### DISCUSSION THREAD

**Discussion by sdpatil07** [GreenleafLab/ArchR#1801] [Questions / Documentation]
Hi,
i am getting following error for addArchRLocking function:

> addArchRLocking(locking = TRUE)

Error in addArchRLocking(locking = TRUE) : 

  could not find function "addArchRLocking"

i have installed ArchR just last week, so i think it should be upto date. Could you please suggest what i should do?  Thanks in advance.

**Comment by rcorces** [comment]
that is only available on the `dev` branch currently

**Reply by sdpatil07** [reply]
ohh thanks. Could you please tell me whats this function is for? i couldn't find any description in https://github.com/rcorces/ArchR_Website_Testing/blob/master/bookdown/21_MultiomicDataAnalysis.Rmd

**Reply by rcorces** [reply]
the documentation for this function has not been released but will be soon. the website you linked is the dev prototype of the to-be-released documentation.
